### ISSUE TITLE
Sync before giving Chrome reminder-to-track notification 

### ISSUE THREAD

**Issue by wardweistra** [toggl/toggl-button#723]
Please follow this guide when creating a new issue. This speeds up the process of replicating the issue and finding a solution.

### Environment details

* Operation system: Mac Sierra 10.12.1 (16B2555)
* Extension version: 1.0.18

### Expected behavior
Chrome extension should know when timer is already running, before giving notification of time track reminder.

### Actual behavior
When timer is running (I also use Mac Toggl application to start the timer) Chrome extension will still popup reminder to track. Only after opening the extension it will sync with Toggl backend, learn that the timer was already running and display as such.
Could you do a sync to check whether timer is already running just before popping up the reminder notification? Currently it gives me unnecessary popups.

### Steps to reproduce the behavior
Install both Mac and Chrome Toggl apps. Start timer on Mac app. See that Chrome extension doesn't reflect that a timer was started. Wait until Chrome extension gives reminder to track time.

### Additional details
Love both apps!

**Comment by lee-tts** [comment]
This old problem seems to still exist. I am on Chromium / Ubuntu 16.04 and the extension frequently is not in sync with the Toggl website. Only once you click on the extension's icon does it sync with the site's status and start showing the time, etc.

**Comment by tcrammond** [comment]
We should have this check implemented now. Please let us know if there are further problems 🙇 .

**Comment by Schnouki** [comment]
@tcrammond Sorry but the fix doesn't seem to be working for me. I just had a reminder-to-track notification a few minutes ago, even though my timer is running (webapp).

Using Firefox 68.0b12 (Linux x86_64) and Toggl Button 1.37.4.
